jupyter-pystata/Dockerfile

```diff
@@ -29,4 +26,25 @@ COPY jupyter-pystata/resources/copy_license_on_startup.py \
 	/home/$NB_USER/.ipython/profile_default/startup/copy_license_on_startup.py
```

Comment:
If the license copy is performed in the first cell of the notebook then the script for `copy_license_on_startup.py` is not required.